rpcs3/evdev_joystick_handler.cpp

```diff
@@ -168,6 +168,30 @@ void evdev_joystick_handler::Close()
     }
 }
 
+inline int Deadzone(f32 input)
+{
+	int	deadzone=g_evdev_joystick_config.deadzone;
+	float deadzone_a=128.0/(128.0-float(deadzone));
```

Comment:
I'm pretty sure a lot (all?) of these casts to floats aren't needed. A float divided by an int produces a float.